ruby/cross_service_examples/item_tracker/README.md

```diff
@@ -0,0 +1,113 @@
+# Deploy a sample application using the AWS SDK for Ruby, Amazon Aurora, and Amazon Simple Email Service (SES)
+
+## Overview
+This example code comprises a "real-world" reference application showcasing a [serverless Amazon Aurora database](https://aws.amazon.com/rds/aurora/) using the [AWS SDK for Ruby](https://aws.amazon.com/sdk-for-ruby/).
+
+This code is written to be browsed in an exploratory manner.
+For a more hands-on experience, you can "run" the application by invoking this example code using the [instructions below]((#invoke-this-example-code)).
+
+# ⚠️ Caution
```

Comment:
This section is different from the template. https://github.com/awsdocs/aws-doc-sdk-examples/wiki/single-cross-service-level-README-template#%EF%B8%8F-important 
Which should change?

Reply:
Fixed to comply.